cppsolutions/StockPrice.cpp

```diff
@@ -0,0 +1,52 @@
+#include <vector>
+#include <iostream>
+
+using namespace std;
+
+class StockPrice
+{
+private:
+
+	vector<vector<int>> testCases{
+		{10, 7, 5, 8, 11, 9},
+		{10, 7, 11, 5, 8, 9},
+		{10, 7, 7},
+		{10, 7, 4, 3}
+	};
+
+	vector<int> getInput(int index)
+	{
+		return testCases[index];
+	}
+
+	int getMaxProfit(vector<int> inputArray)
+	{
+		int maxDifference = inputArray[1] - inputArray[0];
```

Comment:
Not required for this code, but I hope you will handle that corner case when stocks array size is less than 2.